cmd/rpcdaemon/cli/config.go

```diff
@@ -557,6 +535,48 @@ func StartRpcServer(ctx context.Context, cfg httpcfg.HttpCfg, rpcAPI []rpc.API)
 	return nil
 }
 
+func startAuthenticatedRpcServer(ctx context.Context, cfg httpcfg.HttpCfg, rpcAPI []rpc.API) error {
+	var engineListener *http.Server
+	var engineSrv *rpc.Server
+	var engineHttpEndpoint string
+	var err error
+
+	log.Trace("TraceRequests = %t\n", cfg.TraceRequests)
+	srv := rpc.NewServer(cfg.RpcBatchConcurrency, cfg.TraceRequests, cfg.RpcStreamingDisable)
+
+	var rpcAPIList []rpc.API
+
+	for _, api := range rpcAPI {
+		if api.Namespace == "engine" || api.Namespace == "eth" {
```

Comment:
This check is unnecessary since we always pass `AuthAPIList()` to this function.

Reply:
Ah right, this was modified before changing the list. Thanks!